setup.py

```diff
@@ -30,7 +30,7 @@
     },
 
     install_requires=[
-        'pylint==1.4.4',
+        'pylint>=1.5.0,<1.6.0',
```

Comment:
@nedbat this just caused problems for me (#24). If specific projects have issues with versions of pylint, those projects should pin the version. Tying this library to a specific version, however, is problematic for those who want to use other versions.